recipes-support/swupdate/swupdate_%.bbappend

```diff
@@ -1,8 +1,4 @@
 # Configure swupdate client/agent for our purposes.
 # In general we only want the minimum functionality
 # required to verify and install an image file.
-FILESEXTRAPATHS_append := "${THISDIR}/${PN}:"
-
-PACKAGECONFIG_CONFARGS = ""
-
-DEPENDS_append += " mtd-utils"
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
Please add back the newline after this line. This will help in the future to distinguish when this line here really was changed (now it wasn't but it doesn't have a following newline, that's why git will show it as change on this line).